src/Stratis.Bitcoin.Features.Consensus/RuleContextExtention.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using NBitcoin;
+using Stratis.Bitcoin.Consensus;
+using Stratis.Bitcoin.Consensus.Rules;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Consensus
+{
+    public class PosRuleContext
+    {
+        public BlockStake BlockStake { get; set; }
+
+        public Money TotalCoinStakeValueIn { get; set; }
+
+        public uint256 HashProofOfStake { get; set; }
+
+        public uint256 TargetProofOfStake { get; set; }
+    }
+
+    public static class RuleContextExtention
```

Comment:
Why extension?

Reply:
yeah good point I will remove this class